packages/special-pages/pages/onboarding/app/components/RiveAnimation.js

```diff
@@ -48,8 +49,9 @@ export function RiveAnimation ({ state, stateMachine, artboard, inputName, isDar
     // handle light/dark mode
     useEffect(() => {
         function handle () {
+            if (!stateMachine) return
             const inputs = rive.current?.stateMachineInputs(stateMachine)
-            const themeInput = inputs?.find(i => i.name === 'Light?')
+            const themeInput = inputs?.find(i => i.name.startsWith('Light'))
```

Comment:
new animations have the name `Light`, whilst others had `Light?` -> so here I'm just allowing both